bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v3/PageImplTest.java

```diff
@@ -89,4 +89,10 @@ protected void testIsClientlibsAsync_undefinedPolicy() throws Exception {
         assertFalse(async);
     }
 
+    @Test
+    protected void testIsDataLayerClientlibIncluded() {
```

Comment:
We should also have a test for skipping the clientlib include with the new property.